storage/innobase/log/log0online.cc

```diff
@@ -1538,10 +1538,10 @@ log_online_open_bitmap_file_read_only(
 	if (srv_data_home_len
 			&& srv_data_home[srv_data_home_len-1]
 			!= SRV_PATH_SEPARATOR) {
-		ut_snprintf(bitmap_file->name, FN_REFLEN, "%s%c%s",
+		ut_snprintf(bitmap_file->name, sizeof(bitmap_file->name), "%s%c%s",
```

Comment:
This is a legitimate fix, but it should be applied to 5.5 branch as well. So, please file a new bug and fix this part in all 3 branches.